pkg/mindpak/bundle.go

```diff
@@ -18,13 +18,18 @@
 package mindpak
 
 import (
+	"archive/tar"
+	"compress/gzip"
 	"crypto/sha256"
 	"fmt"
 	"io"
 	"io/fs"
 	"os"
+	"path/filepath"
 	"regexp"
 	"strings"
+
+	"github.com/spf13/afero"
```

Comment:
For posterity: This library is used instead of go-billy because go-billy does not implement the standard `fs.FS` interfaces. This means that various parts of the code would have to be changed to use billy-specific types, instead of using the standard Go interfaces.